scripts/install/opt/seagate/cortx/motr/sanity/cortx_lnet_sanity.sh

```diff
@@ -80,14 +80,14 @@ verify_lnet_conf_exists()
             msg "LNET conf file found here $LNET_CONF_FILE is not empty."
         fi
     fi
-    return $RESULT
+    return "$RESULT"
 }
 
 verify_lnet_conf_data()
 {
-    local RESULT=$ERR_LNET_BAD_CONF_DATA
+    local RESULT=${ERR_LNET_BAD_CONF_DATA}
```

Comment:
`$ERR_LNET_BAD_CONF_DATA` and `${ERR_LNET_BAD_CONF_DATA}` are two different ways to reference a variable.

What is the reason to change this?
If you want to change this, we need to change all places. But this patch only changes some of them.

Reply:
And the commit message is to remove the "double quoting" problem.
this kind of error/warning is not mentioned in the commit log.
Is this a real warning or error?

Reply:
> `$ERR_LNET_BAD_CONF_DATA` and `${ERR_LNET_BAD_CONF_DATA}` are two different ways to reference a variable.
> 
> What is the reason to change this? If you want to change this, we need to change all places. But this patch only changes some of them.

Changed as per suggestion

Reply:
> And the commit message is to remove the "double quoting" problem. this kind of error/warning is not mentioned in the commit log. Is this a real warning or error?

Updated the statement with other issues as well. Thanks